pkg/pod/envresolve/env.go

```diff
@@ -71,6 +72,7 @@ func getResourceFieldRef(from *kapi.EnvVarSource, c *kapi.Container) (string, er
 }
 
 // GenEnvVarRefValue returns the value referenced by the supplied EnvVarSource given the other supplied information
+// TODO: how to handle this? It is also used by the build controller which deals with internal versions
```

Comment:
@mfojtik 

The set/env command needs this helper to receive a corev1.EnvVarSource and a corev1.Container.
The build controller depends on a separate helper in pkg/build/controller/common which depends on the current (internal) signature of this helper